src-client/global-shims.d.ts

```diff
@@ -6,6 +6,8 @@ interface Window {
   atomVars: {
     home: ResolvablePromise<string>
     numberEqns: ResolvablePromise<boolean>
+    mjxTeXExtensions: ResolvablePromise<string[]>
+    mjxUndefinedFamily: ResolvablePromise<string[]>
```

Comment:
Probably could get rid of a bit of redundancy if instead of three `ResolvablePromise`s we had just one:
```ts
mathJaxConfig: ResolvablePromise<MathJaxConfig>
// ... 
interface MathJaxConfig {
numberEquations: boolean
mjxTeXExtensions: string[]
mjxUndefinedFamily: string
}
```

Could just pass `atomConfig().mathConfig` directly through in that case.